index.js

```diff
@@ -1,7 +1,20 @@
 const readline = require('readline-sync');
+const calc = require('./calculator');
 
 console.log('Welcome to the calculator!');
 
-console.log('Please enter some input:');
-const response = readline.prompt();
-console.log('You entered: ' + response);
+console.log('Enter the operator:');
+const operator = readline.prompt();
+
+console.log(`How many numbers do you want to ${operator} ?`);
+const c = parseInt(readline.prompt());
+
+const calculator = calc.create_calculator(operator);
+
+for (var i = 0; i < c; i++) {
+  console.log('Enter a number:');
+  const n = parseInt(readline.prompt());
```

Comment:
Not a bug exactly... but why parseInt here? What if the user wants to add 1/3 to 1/2? All numbers in JS are floats by default, so it seems odd to force a cast to int here.

Reply:
If however we decide we *do* want to do this... then we should handle the error case of when the user's input cannot be parsed as an int. 

From the docs (linked below) we can see parseInt returns NaN if it can't parse the input. If that happens we probably don't want to pass it on to the calculator. Raise an exception or at least alert the user at this point. Whichever approach you take, add a unit test to lock in the desired behaviour. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt